app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
     pass
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+    Person.people = {person.name: person for person in person_list}
+
+    for i in range(len(person_list)):
```

Comment:
instead of raw indexes you can just loop in this way:
`for person in person_list`
In this way you instantly get user dict object